x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/tabs/index.tsx

```diff
@@ -194,7 +186,6 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
         <LazyTimelineTabRenderer
           timelineId={timelineId}
           shouldShowTab={isGraphOrNotesTabs}
-          isOverflowYScroll={activeTimelineTab === TimelineTabs.session}
           dataTestSubj={`timeline-tab-content-${TimelineTabs.graph}-${TimelineTabs.notes}`}
         >
           {isGraphOrNotesTabs ? getTab(activeTimelineTab) : null}
```

Comment:
I think once analyzer is cleaned up, we should test old timeline links that direct to session view/analyzer. Currently if `activeTab` in the url is `session`, it still goes to this line and renders null. Probably should open the graph in flyout, or default to query tab.

![image](https://github.com/user-attachments/assets/918ede31-8395-4a0d-908a-4c73dc98b100)

Reply:
Good point. We should have done this months ago when we enabled the advanced setting by default 😂
I can add some logic here and test it!